lock-api/src/test/java/com/palantir/lock/client/UnreliableTimelockClientTest.java

```diff
@@ -171,9 +186,30 @@ public void verifyCurrentTimeMillisCallsDelegate() {
         verify(timeLockClient).currentTimeMillis();
     }
 
+    private UnreliableTimeLockService createAlwaysBuggyTimelockService() {
+        return createTimelockService(createAlwaysBuggyFactory());
+    }
+
+    private UnreliableTimeLockService createTimelockService(BuggifyFactory factory) {
+        return new UnreliableTimeLockService(timeLockClient, timestampManager, factory);
+    }
+
     private static BuggifyFactory createAlwaysBuggyFactory() {
         BuggifyFactory factory = mock(BuggifyFactory.class);
         when(factory.maybe(anyDouble())).thenReturn(DefaultBuggify.INSTANCE);
         return factory;
     }
+
+    // Visible as it's used as a method source
+    static Stream<Named<Consumer<UnreliableTimeLockService>>> timestampMethods() {
```

Comment:
non-actionable: This worked on my machine with a private modifier, it might mean I am on a more recent JDK.